### PR TITLE
feat(annotations): Improve annotation activity card interactions

### PR DESCRIPTION
**Pull request by jstoffan**
The activity feed is in kind of an awkward place with regard to code conventions. I tried to change/refactor only what was helpful to this particular enhancement rather than the entire component, so it's a bit of mashup at the moment. I'm open to feedback on it, though.

Todo
- [x] Design review

### CODE REVIEW

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 1]
```diff
@@ -28,82 +32,76 @@
...
+                    animation-fill-mode: forwards;
                 }
             }
         }
 
-        .bcs-activity-feed-annotation-activity.bcs-is-focused {
-            padding-left: 21px;
-            border-left: 4px solid $bdl-yellorange;
+        .bcs-version-item {
+            margin-top: $bdl-grid-unit * 3;
```
Comment (ConradJChan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:72]: The intended margin is 16px, may want to run this by design

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 2]
```diff
@@ -28,82 +32,76 @@
...
+            margin-bottom: $bdl-grid-unit * 3;
```
Comment (ConradJChan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:73]: Did we want to maintain or come up with a new rule to apply a margin-top to the `. bcs-activity-feed-annotation-activity` when it follows a `.bcs-version-item`?

File: src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityItem.test.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import ActivityItem from '../ActivityItem';
+
+describe('src/elements/content-sidebar/activity-feed/activity-feed/ActivityItem', () => {
+    const getWrapper = (props = {}): ShallowWrapper => shallow(<ActivityItem {...props}>Test</ActivityItem>);
+
+    test('should have a default class and passhthrough its className prop', () => {
```
Comment (ConradJChan) [inline src/elements/content-sidebar/activity-feed/activity-feed/__tests__/ActivityItem.test.tsx:8]: `passhthrough`

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss (hunk 1 above)
Comment (jstoffan) [reply src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:72]: Nice catch, this is my bad. I think I was looking at an outdated reference. Will update.

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 4]
```diff
@@ -28,82 +32,74 @@
...
-        .bcs-activity-feed-annotation-activity.bcs-is-focused {
-            padding-left: 21px;
-            border-left: 4px solid $bdl-yellorange;
+        .bcs-version-item {
+            padding-top: 0;
+            padding-bottom: 0;
+            box-shadow: none;
+
+            &,
+            & + .bcs-ActivityItem {
```
Comment (ConradJChan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:73]: Yeah, I was on the fence about this one, but I think it is better from a consistency perspective. Tasks also can be deeplinked and briefly show a focused state so it's probably better to have the margin-top as well

File: src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss [hunk 5]
```diff
@@ -1,5 +1,17 @@
 @import '../../../common/variables';
 
+@keyframes ba-ActivityItem-callout {
```
Comment (ConradJChan) [inline src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss:3]: `ba-` namespace intentional?

**Comment by ConradJChan** [comment]
Really love the animation, pretty slick 💯

**Pull request merged**